src/card.tsx

```diff
@@ -495,6 +510,163 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
     }
 );
 
+export const NakedCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
+    (
+        {
+            media,
+            headline,
+            pretitle,
+            pretitleLinesMax,
+            subtitle,
+            subtitleLinesMax,
+            title,
+            titleLinesMax,
+            description,
+            descriptionLinesMax,
+            extra,
+            actions,
+            button,
+            buttonLink,
+            dataAttributes,
+            'aria-label': ariaLabel,
+            onClose,
+            ...touchableProps
+        },
+        ref
+    ) => {
+        const isTouchable = touchableProps.href || touchableProps.to || touchableProps.onPress;
+        const isCircularMedia = media && media.type === Image && (media.props as any).circular;
+
+        return (
+            <CardContainer
+                ref={ref}
+                dataAttributes={{'component-name': 'NakedCard', ...dataAttributes}}
+                aria-label={ariaLabel}
+                className={isTouchable ? styles.touchableContainer : undefined}
+            >
+                <BaseTouchable maybe {...touchableProps} className={styles.touchable} aria-label={ariaLabel}>
+                    <div className={styles.mediaCard}>
+                        <div style={{position: 'relative'}}>
+                            {isTouchable && (
+                                <div
+                                    className={classNames(styles.touchableMediaOverlay, {
```

Comment:
The media of the naked differs from the media hover definition, and uses the same overlay that displayMedia and poster cards with the following tokens:

`coverBackgroundHover`, `coverBackgroundPressed`

Reply:
fixed, looks much better now!